### PR TITLE
Document memory profiling options

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

File: site/source/docs/porting/Debugging.rst [hunk 1]
```diff
@@ -250,7 +250,50 @@ If your code hits an infinite loop, one easy way to find the problem code is to
...
+usage inside that object, so you need other tools for that, which we will now
+describe.
+
+Emscripten supports
+`mallinfo() <https://man7.org/linux/man-pages/man3/mallinfo.3.html>`_, which lets
+you get information from ``dlmalloc`` about current allocations. For example
+usage, see
+`the test <https://github.com/emscripten-core/emscripten/blob/9bb322f8a7ee89d6ac67e828b9c7a7022ddf8de2/tests/mallinfo.cpp>`_.
+
+Emscripten also has a ``--memoryprofiler`` option that the displays memory usage
```
Comment (haberbyte) [inline site/source/docs/porting/Debugging.rst:283]: the displays -> displays

Comment (kripken) [inline site/source/docs/porting/Debugging.rst:283]:
```suggestion
Emscripten also has a ``--memoryprofiler`` option that displays memory usage
```

Comment (kripken) [reply site/source/docs/porting/Debugging.rst:283]: Thanks @haberbyte !

File: site/source/docs/porting/Debugging.rst [hunk 2]
```diff
@@ -250,7 +250,50 @@ If your code hits an infinite loop, one easy way to find the problem code is to
...
 
+.. _debugging-profiling:
 
+Profiling
+=========
+
+Speed
+-----
+
+To profile your code for speed, build with ``--profiling``, then run the code in
```
Comment (dschuff) [inline site/source/docs/porting/Debugging.rst:261]: Is this option documented elsewhere? Could we add a link?

Comment (kripken) [reply site/source/docs/porting/Debugging.rst:261]: Yes, the `emcc --help` page has info. I added a link now.

**Review by dschuff — APPROVED** [review]

**Pull request merged**